docs/framework/dataframe.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: "Dataframe"
+---
+
+**writer framework places the dataframe at the core of the application**. This is a great way for modeling a complex and massive data system.
+it offers components as `dataframe` to manipulate dataframes. These components allow you to visualize and interact with dataframes.
+
+| compatibility      | dataframe                             |
+|--------------------|---------------------------------------|
+| `pandas.DataFrame` | x                                     |
+| `polar.DataFrame`  | x                                     |
+| `list of records`  | x (with `EditableDataframe`)          |
+
+### Use a dataframe
```

Comment:
The base headline level should be 2 as I see for other documents

https://github.com/writer/writer-framework/blob/97a5948a8afb68e5c10ffd4797cbc049f55125bb/docs/framework/ai-module.mdx?plain=1#L7